R/rd-data.R

```diff
@@ -134,7 +134,26 @@ as_data.tag_examples <- function(x, path, ...,
     code_env <- new.env(parent = env)
     code_env$not_run <- function(...) invisible()
 
-    expr <- evaluate::evaluate(text, code_env, new_device = TRUE)
+    output_handler <- evaluate::new_output_handler(
+      value = function(x) {
+        if (isS4(x)) {
```

Comment:
Good point. It looks like the `knit_print.default` calls `knitr:::normal_print()` which already took care of this.